### PR TITLE
Adding Rb-webrick

### PR DESCRIPTION
**Pull request by d-cunningham**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by d-cunningham** [comment]
@conda-forge/staged-recipes, recipe ready for review

File: recipes/r-webrick/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: 87e9b8e39947b7925338a5eb55427b11ce1f2b25a3645770ec9f39d8ebdb8cb4 
+
+build:
+  noarch: generic
```
Comment (BastianZim) [inline recipes/r-webrick/meta.yaml:13]: If you have a skip, it cannot be platform independent.

File: recipes/r-webrick/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies -n $PREFIX/bin {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
+  skip: {{ win }}
```
Comment (BastianZim) [inline recipes/r-webrick/meta.yaml:18]:
```suggestion
  skip: True  # [win]
```

**Review by BastianZim — CHANGES_REQUESTED** [review]
@d-cunningham Apologies for no review so far but we currently have a very small ruby review team. I have added some small comments and requested a review from the whole team.

**Pull request closed without merging**